isisd/isis_tlvs.c

```diff
@@ -1382,7 +1408,7 @@ static void format_item_ext_subtlvs(struct isis_ext_subtlvs *exts,
 	if (IS_SUBTLV(exts, EXT_SRV6_LAN_ENDX_SID)) {
 		struct isis_srv6_lan_endx_sid_subtlv *lan;
 		if (json) {
-			struct json_object *arr_adj_json, *flags_json;
+			struct json_object *arr_adj_json, *srv6_end_sid_json;
```

Comment:
Ditto

Change to `srv6_endx_sid_json` or `srv6_end_x_sid_json`?

Reply:
Done. I've set srv6_lan_endx_sid_json